src/lib.rs

```diff
@@ -131,6 +137,18 @@ pub fn with<T: AsRef<OsStr>>(path: T, app: impl Into<String>) -> io::Result<()>
     os::with(path, app)
 }
 
+/// Get command that opens path with the default application.
```

Comment:
I think it should be mentioned that this is a 'no-run' sibling of `open::that(…)`, as opposed to, say, `open::with(…)`.